crates/knitwit/src/lib.rs

```diff
@@ -49,12 +44,13 @@ impl Guest for KnitWit {
 
         merge_worlds(&mut resolve, base_world, worlds).map_err(|e| e.to_string())?;
 
-        resolve_to_readable_wit(resolve, id, output_dir).map_err(|e| e.to_string())
+        resolve_to_readable_wit(&resolve, id).map_err(|e| e.to_string())
     }
 }
 
 fn merge_wits(resolve: &mut Resolve, wit_paths: Vec<String>) -> anyhow::Result<()> {
     for path in wit_paths {
+        println!("Merging wit: {}", path);
```

Comment:
Curious if this is user actionable / useful?

Reply:
Good catch! That comes from my debugging efforts.